src/intTest/resources/outbound_uat_data/acceptance/type2-first-mandatory_1.edifact.dat

```diff
@@ -1,7 +1,7 @@
-UNB+UNOA:2+TES6+YY21+200610:1438+00000001'
+UNB+UNOA:2+TES6+XX11+200610:1438+00000001'
```

Comment:
change was needed because of src/intTest/resources/application.yml

Reply:
I'd add mapping for this instead of changing it

Reply:
right, YY21 mapping added